apps/webpack.js

```diff
@@ -44,6 +44,9 @@ var baseConfig = {
       },
     ],
   },
+  node: {
+    fs: 'empty',
```

Comment:
`typo-js` uses the `fs` module, and webpack chokes on that unless I add this to our webpack config.